Edit Logo for Table of Content

JuanDiegoGuerra opened the pull request:
Adding my esports organization (**ArchEsports**) logo in the **_img_** folder and replacing the image in the README.md file 😄

JuanDiegoGuerra committed:
Edit Logo for Table of Content

Adding my esports org logo in the img folder and replacing the image in the README.md file

JuanDiegoGuerra committed:
Update .gitignore

Adding to the list in order to fix linters:
.stylelinetrc.json
.hintrc
yarn.lock

JuanDiegoGuerra committed:
Add 2 files for linters - Update Gitignore

adding .hintrc and .stylelintrc.json

JuanDiegoGuerra committed:
Update styles.css

Updating styles.css in order to work well with linter

JuanDiegoGuerra commented:
The **Stylelint Linter** request seems to be a sort of bug cause I dont know why it says: _"Expected indentation of 2 spaces"_
and the **Lighthouse** one is always waiting for browser and I don't know what that means.

Shedrack-Sunday commented on README.md:
```diff
@@ -1,7 +1,7 @@
 <a name="readme-top"></a>
 
 <div align="center">
-  <img src="img/fbprofilepic.jpg" alt="logo" width="50"  height="auto" />
+  <img src="img/archesportslogo.jpg" alt="logo" width="50"  height="auto" />
   <br/>
 </div>
```
- [ ] Nice work adding some of the sections on the [README](https://github.com/microverseinc/readme-template), however, you're meant to add all sections with just `FAQ` being optional. I've attached screenshots below to show you what to add. This makes your project possess all sections that make it look professional 

## Expected README  Header:
![image](https://user-images.githubusercontent.com/50990925/226778810-21f02d49-388c-44c6-bdca-bb8f80c51658.png)

Shedrack-Sunday reviewed (changes requested):
Hi @JuanDiegoGuerra       👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 
![](https://user-images.githubusercontent.com/80612925/157397398-00904e26-60ec-46cd-9510-240661c250ef.gif)


Highlights
✅All linters are checked
✅ Proper use of .gitgnore
  
# Required Changes ♻️

- [x] I must admit you have really put in the effort so far in this project, however, the changes required below needs to be implemented:

- Please make sure all linters are passing
- Your `MIT` License seems to be broken, kindly fix this 👍 
- I highlighted some sections of your README to be added. Kindly check the comments and [Template](https://github.com/microverseinc/readme-template) provided

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

JuanDiegoGuerra committed:
Update .gitignore

Deleting some names from the list.

JuanDiegoGuerra committed:
Add ".gitattributes" and "LICENSE" files

Looks like somehow I deleted these 2 files that were important for the project and I'm restoring them.

JuanDiegoGuerra committed:
Update README.md

Adding all the information requiered in the PR :smile: .

JuanDiegoGuerra committed:
Update styles.css

Fixed indentation error for stylelint filter

JuanDiegoGuerra committed:
Update linters.yml

Changing node version for lighthouse in order to fix the error.

JuanDiegoGuerra committed:
Update README.md

Adding all the information needed for the README file, including FAQ.

JuanDiegoGuerra commented:
@Shedrack-Sunday Hello! I just finished everything 🤩. It was kinda hard working for me. It wasn't hard at all but I think there was a lot of information I had to review in order to complete it. Hope I fixed everything you tell me to. I'll be waiting for your feedback. Thanks again!
👉🏼All Linters are passing: ✅
![Success](https://user-images.githubusercontent.com/45726631/226812805-ed6033f8-ee4d-45d3-a4dc-367b16b4a0b6.JPG)
♻Fixing the LICENSE (it wasn't there for some reason.) ✅
![LICENSE](https://user-images.githubusercontent.com/45726631/226813519-48d9edc0-9be1-47b6-a5f7-ee8e75db77eb.JPG)
✏Last commit to add all the information needed in the README file: ✅
![UpdateREADME](https://user-images.githubusercontent.com/45726631/226813258-6c7ca061-f97c-4f47-9e51-1fb53848015f.JPG)

Shedrack-Sunday reviewed (approved):
# STATUS: APPROVED ♻️♻️
Hi @JuanDiegoGuerra 

To highlight 👍🏽
✔️ Proper use of semantics
✔️ No linter errors

Your project is complete! There is nothing else to say other than... it's time to merge it 🚀
Congratulations! 🎉

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

shella12 reviewed (commented):
Hi @JuanDiegoGuerra , 

#  Status: Invalidated :red_circle: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has wrong project link. You are submitting" Hello Microverse" project link for "portfolio" project. Please make sure to submit the correct link on to dashboard.


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._
